src/main/resources/static/css/styles.css

```
@@ -89,21 +279,55 @@ li {
-    padding: 20px 30px;
+    background: transparent;
+    color: rgba(var(--base-color-white), 0.9);
+    padding: 40px 30px;
     text-align: center;
-    color: #666;
-    border-top: 1px solid #ddd;
+    border-radius: 10px;
+    border-top: solid #f7f7f717;
+    border-width: 1px;
+
+    @supports (corner-shape: squircle) {
```

Biome (2.3.14)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)